tcf_website/static/common/sorting.js

```diff
@@ -20,6 +20,38 @@ function cmpByProp(prop, asc) {
         if (prop === "date") { // convert date string to int
             valA = dateToInt(valA);
             valB = dateToInt(valB);
+        } else if (prop === "recency") {
+            try {
+                const sessionA = valA.match(/^[a-zA-Z]+/)[0];
+                valA = valA.match(/-?[0-9]\d*(\.\d+)?/)[0];

```

Comment:
I'm like 90% sure that what you're doing here is setting `sessionA` to the season and `valA` to the year, but it might be more descriptive to first assign the year value to an intermediate variable (e.g. `yearA`) and then combine it all into `valA` at the end for readability.

Reply:
Yep, I just fixed this within getSessionNum()